tools/importers/CouchDB_MRI_Importer.php

```diff
@@ -11,8 +11,11 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://www.github.com/aces/Loris/
  */
-require_once __DIR__ . "/../vendor/autoload.php";
-require_once 'generic_includes.php';
+require_once __DIR__ . '/../../vendor/autoload.php';
+require_once __DIR__ . '/../generic_includes.php';
```

Comment:
requiring is a side-effect, it should be in the other file not the class definition file.